### DISCUSSION TITLE
sitemap problem

### DISCUSSION THREAD

**Discussion by benthien** [foundation/proton#19] [General]
The build process works properly on two sites I have tested. As mentioned previously, I get an error when I run proton watch even though I have followed the procedures outlined in the beta 5 release. One of my sites has a number of subdirectories in pages. The sitemap produced has \ instead of / for the files in these subdirectories. For example,
**`<loc>https://siteName/subDurectory\file.html`**

**Comment by joeworkman** [comment]
You are on Windows?

**Reply by benthien** [reply]
Yes. I am on Windows 11

**Reply by benthien** [reply]
I set up proton on Linux under windows wsl. Sitemap was correct. The two sites I tried ran very fast on wsl as long as I set `pretty: false` in proton.xml. If pretty was set to true, proton build ran but was extremely slow. I still get the same error as before when I run` proton watch`.
